Source/WebKit/WebProcess/GPU/graphics/ImageBufferShareableBitmapBackend.h

```diff
@@ -38,7 +43,13 @@ namespace WebKit {
 
 class ShareableBitmap;
 
-class ImageBufferShareableBitmapBackend final : public WebCore::PlatformImageBufferBackend, public ImageBufferBackendHandleSharing {
+#if USE(CG)
+using ImageBufferShareableBitmapBackendBase = WebCore::ImageBufferCGBackend;
+#elif USE(CAIRO)
+using ImageBufferShareableBitmapBackendBase = WebCore::ImageBufferCairoBackend;
+#endif
```

Comment:
This is a little bit awkward. 

1. Both `ImageBufferCGBackend` and `ImageBufferCairoBackend` are WebCore objects. So why do we need to have them redefined as `ImageBufferShareableBitmapBackendBase` in WebKit?
2. Both do not handle any backend sharing.
3. `ImageBufferCGBackend` is not necessarily unaccelerated. It the base of `ImageBufferIOSurfaceBackend` also.

Can't we keep them in WebCore under the name ImageBufferPlatformBackend?